subprojects/tooling-api/src/testFixtures/groovy/org/gradle/integtests/tooling/fixture/CrossVersionTestEngine.java

```diff
@@ -280,7 +280,16 @@ private ClassLoader toolingApiClassLoaderForTest(Class<?> testClass) {
 
     private ToolingApiDistribution getToolingApi(String versionToTestAgainst) {
         if (toolingApi == null) {
-            toolingApi = new ToolingApiDistributionResolver().withDefaultRepository().resolve(versionToTestAgainst);
+            // simplified try-with-resources equivalent, because Java 6
```

Comment:
Interesting, I thought we build all tests and test fixtures with Java 8, even when the main source set is on Java 6. Seems like this isn't the case, yet. Something for another PR I suppose.